pages/help.js

```diff
@@ -79,11 +80,26 @@ export default function Help() {
             local exam session.
           </li>
           <li>
-              <a href="https://apps.fcc.gov/cores/userLogin.do?csfrToken=" className="underline">
-                FCC FRN Registration
-              </a>{" "}
-              - Once you finish studying, you will need to get an FRN from the FCC in order to start the test in the first place.
-              You will need to register with the FCC in order to do this!
+            <a
+              href="https://apps.fcc.gov/cores/userLogin.do?csfrToken="
+              className="underline"
+            >
+              FCC FRN Registration
+            </a>{" "}
```

Comment:
weird formatting here, pls fix! I would suggest running prettier on your code before PRing.

Reply:
this is what Prettier did